astropy/io/fits/time.py

```diff
@@ -0,0 +1,174 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from collections import OrderedDict
+
+import numpy as np
+
+from . import Header, Card
+
+from ...coordinates import EarthLocation
+from ...table import Table, QTable, Column
+from ...time import Time
+from ...time.formats import FITS_DEPRECATED_SCALES
+from ...units import Quantity
+
+
+class FitsTime(object):
+    """
+    A class to maintain the metadata of time columns stored in a FITS binary table
+    (header and data) and to convert those time columns into `~astropy.time.Time` columns.
+
+    Parameters
+    ----------
+    nfields : int
+        Number of fields(columns) in the binary table.
+    """
+
+    def __init__(self, nfields):
+        """
+        Set default time metadata.
+        """
+        # Default time scale UTC
+        self.scale = 'UTC'
```

Comment:
OK, having a placeholder like that would make sense. We can return to this once we try some real files...